src/util/isArray.ts

```diff
@@ -1 +1 @@
-export const isArray = Array.isArray || (x => x && typeof x.length === 'number');
\ No newline at end of file
+export const isArray = Array.isArray || (<T>(x: any): x is T[] => x && typeof x.length === 'number');
```

Comment:
`x is T[]` lets the compiler know that if the method returns true, than the `typeof x` is `T[]`